File: assignment 10 - C# practice problems/js/6.js
//Write a JavaScript program to find the largest of three given integers. 
function largestOfThreeDriverFunction() {
    let numbers = prompt("Enter three Integers seperated by a space");

    if (numbers != null) {
        numbers = numbers.trim();
        numbers = numbers.split(" ");

        let num1 = parseFloat(numbers[0]);
        let num2 = parseFloat(numbers[1]);
        let num3 = parseFloat(numbers[2]);

        // if (num1 > num2) {
        //     if (num1 > num3) {
        //         alert(num1 + " is greater");
        //     } else {
        //         alert(num3 + " is greater");
        //     }
        // } else if (num2 > num3) {
        //     alert(num2 + " is greater");
        // } else {
        //     alert(num3 + " is greater");
        // }

        alert(Math.max(num1, num2, num3) + " is greater");
    }
}
largestOfThreeDriverFunction()

File: assignment 10 - C# practice problems/6.cs
//Write a c# program to find the largest of three given integers. 

using System;
namespace MyNameSpace
{
    class Program6
    {
        public static void LargestOfThreeDriverFunction()
        {

            Console.WriteLine("Enter three Integers:");
            double num1 = double.Parse(Console.ReadLine());
            double num2 = double.Parse(Console.ReadLine());
            double num3 = double.Parse(Console.ReadLine());


            if (num1 > num2)
            {
                if (num1 > num3)
                {
                    Console.WriteLine(num1 + " is greater");
                }
                else
                {
                    Console.WriteLine(num3 + " is greater");
                }
            }
            else if (num2 > num3)
            {
                Console.WriteLine(num2 + " is greater");
            }
            else
            {
                Console.WriteLine(num3 + " is greater");
            }

        }
    }
}
